numbergame.py

```diff
@@ -43,8 +58,12 @@ def score(integer): #This is how it calculates which message to give
         print("That was pretty bad... It sadly took you", numberOfTries, "tries.")
     else:
         print("I'm sorry about this, but I have to say the truth. You were terrible. It took you", numberOfTries, "tries.")
+    if LOLZ:
+        sys.exit(255)

```

Comment:
What does this do

Reply:
because if they got it right they probably cheated :P

Reply:
change it if you want